documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -841,10 +841,11 @@ include::{testDir}/example/ParameterizedTestDemo.java[tags=EnumSource_regex_exam
 [[writing-tests-parameterized-tests-sources-MethodSource]]
 ===== @MethodSource
 
-`@MethodSource` allows you to refer to one or more _factory_ methods of the test class.
-Such methods must return a `Stream`, `Iterable`, `Iterator`, or array of arguments. In
-addition, such methods must not accept any arguments. By default such methods must be
-`static` unless the test class is annotated with `@TestInstance(Lifecycle.PER_CLASS)`.
+`@MethodSource` allows you to refer to one or more _factory_ methods of the test or
+external class. Such methods must return a `Stream`, `Iterable`, `Iterator`, or array
+of arguments. In addition, such methods must not accept any arguments. By default such
+methods must be `static` unless the test class is annotated with
+`@TestInstance(Lifecycle.PER_CLASS)`.
```

Comment:
The last sentence is no longer technically correct.

Factory methods in external classes must always be `static`.

Thus, please reword that to take into account the change in this PR.